pages/index.tsx

```diff
@@ -54,6 +59,19 @@ export default function Home() {
         </div>
       </main>
 
+      <div>
+        <h1 className="text-center text-2xl font-bold">New Users</h1>
+        <div className="flex flex-wrap w-full mx-auto justify-center">
+          {Array.from({ length: NUM_MEMBERS }).map((_, i) => (
+            <NewMembersCard
+              key={i}
```

Comment:
Key shouldn't be index here

Reply:
It is temporary for rendering the dummy data ,